### PR TITLE
Structured pattern corrections - Trusted Committer

### PR DESCRIPTION
**Pull request by doronkatz**
Minor corrections and rewording. Additional active/passive voice changes.

### CODE REVIEW

File: patterns/2-structured/trusted-committer.md [hunk 1]
```diff
@@ -75,50 +73,49 @@ README.  As an example:
...
+manager, if there are any issues, listen and empathize to try and understand
+what is preventing the Trusted Committer from being successful. Always
+[thank the Trusted Committer for their continued effort][praise] in making the project successful and set a new date to check-in.
 
 ### Sunsetting a Trusted Committer
 
-There comes a time when removal of a Trusted Committer is necessary, for
-example:
+There are times which necessitate removing a Trusted Committer, such as if the Trusted
+Committer:
```
Comment (spier) [inline patterns/2-structured/trusted-committer.md]:
```suggestion
Committer is:
```

If I change it like this, I would be able to read it as a full sentence.
Is that better?

File: patterns/2-structured/trusted-committer.md [hunk 2]
```diff
@@ -6,51 +6,49 @@ Trusted Committer
...
-members and outlines the role for future candidates.
+What a Trusted Committer handles is up to each project and its maintainers. Ensure you document within the project the scope of your Trusted Committer role. Clear documentation sets expectations for new community members and establishes the role for future candidates.
 
-Below we've provided a few guidelines for what Trusted Committers can be
-invited to do:
+The following are a few guidelines for identifying a potential Trusted Committer:
 
-* If a candidate participates often in community channels (e.g. Slack, JIRA issue triaging, etc.) then becoming a Trusted Committer formalizes their role in community support.
-* A good candidate for a Trusted Committer, is someone who frequently submits code, documentation, or other repository changes.  Start by including this person on pull requests. If they are actively engaging in pull requests, consider approaching them about opportunities for further collaboration on the project.
+* An active participant in community channels (e.g., Slack, JIRA issue triaging, etc.) becomes a Trusted Committer, thereby formalizing their role in community support.
```
Comment (spier) [inline patterns/2-structured/trusted-committer.md]:
```suggestion
* An active participant in community channels (Slack, JIRA issue triaging, etc.) becomes a Trusted Committer, thereby formalizing their role in community support.
```

I am sure the form "e.g., " is grammatically correct, however I find it hard to read.

As the "e.g." and the "etc" are both saying kind of the same, I propose to remove one of them.

File: patterns/2-structured/trusted-committer.md [hunk 3]
```diff
@@ -6,51 +6,49 @@ Trusted Committer
...
+In these situations, project maintainers seek ways to recognize and reward the
 work of the contributor above and beyond single contributions.
 
 ## Problem
 
 - Project maintainers want to find ways to scale their ability to support a project
 - Project maintainers want to find ways to lengthen the value delivered by a project
 - Project maintainers want to visibly reward frequent contributors and empower them to amplify their value contribution.
-- Lack of language for recognizing contributions across teams within an organization
+- Lack of mechanism for recognizing contributions across teams within an organization
```
Comment (spier) [inline patterns/2-structured/trusted-committer.md]:
```suggestion
- Lack of mechanism and language for recognizing contributions across teams within an organization
```
 
I agree that the TC concept overall is more of a mechanism, as it comes with expectations, process, etc.
However I do think that the introduction of the term "Trusted Committer" into the shared language of a project is also an essential piece of the change. Therefore I would opt for keeping both.

**Review by spier — CHANGES_REQUESTED** [review]
These are great contributions!
For the most part I find the new language simpler, and the minor content extensions make sense as well.

I left some feedback for your review.

File: patterns/2-structured/trusted-committer.md (hunk 2 above)
Comment (doronkatz) [reply patterns/2-structured/trusted-committer.md]: Totally agree I am for reducing text. In technical writing it is about being as brief as possible. Im sure there are a lot more areas we can do but this is iterative :) Great team work 👍

**Comment by spier** [comment]
Thanks so much for the contribution @doronkatz! 
Your changes are now live in the pattern at https://innersourcecommons.gitbook.io/innersource-patterns/patterns/trusted-committer 🎉

**Pull request merged**